### ISSUE TITLE
Disable resize feature on individual grid items

### ISSUE THREAD

**Issue by amergin** [ManifestWebDesign/angular-gridster#382]
Is it possible to somehow disable the resize feature on certain items on the grid while maintaining the feature on other items?

**Comment by axar** [comment]
you can program it

`for (var h = 0, hl = $dragHandles.length; h < hl; ++h) {
                                if (itemOptions.draggable !== false) {
                                    unifiedInputs[h] = new GridsterTouch($dragHandles[h], mouseDown, mouseMove, mouseUp);
                                    unifiedInputs[h].enable();
                                }
                            }`

`for (var c = 0, l = handlesOpts.length; c < l; c++) {
                if (itemOptions.resizable !== false) {
                    handles.push(new ResizeHandle(handlesOpts[c]));
                }
            }`

you're pretty much adding the if statement. I found this code from one of the branch, I dont remember from where. By doing the above you can specify resizable and draggable properties to false on the gridster item object.

**Comment by PedroF20** [comment]
Where did you add that code?

**Comment by amergin** [comment]
Maybe submit a pull request? It's a hassle to maintain patched libraries.

**Comment by PedroF20** [comment]
True, but the last update this project had was from almost a year ago. I don't there will be any corrections
